### ISSUE TITLE
Setup generic behaviour for Focus Indicator on focused Banner links

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#6711]
Focus indicator punch-out behaviour should be global, therefore this highlighted code below:

![Screen Shot 2020-02-25 at 11 18 43 AM](https://user-images.githubusercontent.com/15113219/75268460-85039600-57c5-11ea-91b2-4635c7532a01.png)

which is currently in these 2 files:
`projects/storefrontstyles/scss/themes/calydon/layout/page-templates/_category-page.scss`
`projects/storefrontstyles/scss/themes/calydon/layout/page-templates/_landing-page-2.scss`

should be **transferred** to this `cx-base` file during the `calydon-as-base` effort
`projects/storefrontstyles/scss/components/content/banner/_banner.scss`

(Note that the **focus indicator** styles should be already available as base in order for this fix to work).

**Comment by developpeurweb** [comment]
Got fixed in GH-6404

**Comment by developpeurweb** [comment]
**For QA:**

- Validate the code was moved
- Validate focus indicator is visible on banners
